core/governance/engine.go

```diff
@@ -370,33 +370,33 @@ func (e *Engine) OnTick(ctx context.Context, t time.Time) ([]*ToEnact, []*VoteCl
 		// this is the new market proposal, and should already be in the slice
 		prop := *ep.ProposalData()
 
-		if prop.Terms.Change.GetTermType() == types.ProposalTermsTypeNewMarket {
-			mktState, err := e.markets.GetMarketState(prop.ID)
-			if err != nil {
-				e.log.Error("could not get state of market %s", logging.String("market-id", prop.ID))
-				continue
-			}
-
+		propType := prop.Terms.Change.GetTermType()
+		if propType == types.ProposalTermsTypeNewMarket || propType == types.ProposalTermsTypeUpdateMarket {
+			mktState, _ := e.markets.GetMarketState(prop.ID)
+			//if err != nil {
```

Comment:
why is that?

Reply:
This was set to have the system tests pass only.
It was not intended to stay.
Corrected now.